src/jsbsim_bridge.cpp

```diff
@@ -210,8 +211,8 @@ void JSBSimBridge::Run() {
   _result = _fdmexec->Run();
 
   std::chrono::duration<double> elapsed_time = current_time - _last_step_time;
-  if (_realtime) {
-    double sleep = _dt - elapsed_time.count();
+  if (_realtime_factor > 0) {
+    double sleep = _dt/_realtime_factor - elapsed_time.count();
```

Comment:
This doesn't look correct: `elapsed_time` contains the sleep time, but you only want to subtract the computation time.

Reply:
Fixed! Thanks for the review!